src/protocol/gossip/impl/remote_subscriptions.cpp

```diff
@@ -10,6 +10,9 @@
 #include <libp2p/protocol/gossip/impl/connectivity.hpp>
 #include <libp2p/protocol/gossip/impl/message_builder.hpp>
 
+#define SPDLOG_TRACE_ON
```

Comment:
The fact you had to define the macro is looking suspicious <img width="16" alt="fry" src="https://user-images.githubusercontent.com/36847043/76315208-4c95a900-62e9-11ea-8ff7-e2ee57c7f70b.png">

Reply:
not sure if needed, or resulted from grasping at straws...

spd was a headache and I had to comment random log lines just to get libp2p to link. I'll put some more time into this.

Reply:
Please check if it works when the macro is not defined.

Reply:
I dropped the commit entirely.